package-lock.json

```diff
@@ -18251,7 +18251,7 @@
     },
     "dev-website-tailwind-config": {
       "version": "git+ssh://git@github.com/merico-dev/dev-website-tailwind-config.git#62017898d43897acc108183cf8313e96e8083b25",
-      "from": "dev-website-tailwind-config@https://github.com/merico-dev/dev-website-tailwind-config"
+      "from": "dev-website-tailwind-config@github:merico-dev/dev-website-tailwind-config"
```

Comment:
Will this affect the website?

Reply:
It came from the `package.json` file, should be fine.